.github/workflows/nightly.yaml

```diff
@@ -97,3 +97,20 @@ jobs:
 
       - name: Test with Gradle
         run: ./gradlew latestDepTest
+
+  issue:
+    name: Open issue on failure
+    needs: [test, testLatestDep]
+    runs-on: ubuntu-latest
```

Comment:
Does this require Ubuntu?

Reply:
Don't know actually. Just copy-paste from examples and it's the same as in all other jobs. Is it important?

Reply:
My impression is JavaScript will work faster than docker but that's it so it's not too important.